cernopendata/modules/fixtures/data/articles/getting-started-with-cms/cms.json

```diff
@@ -21,5 +21,68 @@
       "content": "cms.md",
       "format": "md"
     }
+  },
+  {
```

Comment:
This would add the new articles in the same JSON under the same `getting-started-with-cms` directory.

If we go for the full slug-oriented  approach discussed in https://github.com/cernopendata/opendata.cern.ch/pull/1601#discussion_r149937078 then it would be better to split them up to have:

```
- /articles/geting-started-with-cms-open-data/...
- /articles/cms-guide-for-education/...
- /articles/cms-guide-for-research/...
```

where the directory name would represent the slug of each given article.
```

Reply:
ok! 👍🏻